raiden/tests/scenarios/bf4_non_allowed_ops.yaml

```diff
@@ -0,0 +1,151 @@
+      - serial:
+          name: "Assert that balance changed"
+          tasks:
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 0, state: "opened"}
+      - serial:
+          name: "Node1 tries to withdraw again"
+          tasks:
+            - withdraw: {from: 1, to: 0, total_withdraw: 1_000_000_000_000_000_000, expected_http_status: 409}
+      - serial:
+          name: "Assert that balance did not change"
+          tasks:
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 0, state: "opened"}
+      - serial:
+          name: "Node1 tries to deposit more than it has onchain"
+          tasks:
+            - deposit: {from: 1, to: 0, total_deposit: 101_000_000_000_000_000_000, expected_http_status: 409}
+      - parallel:
+          name: "Assert that balance didn't change"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 0, state: "opened"}
+      - serial: 
+          name: "Close channel"
+          tasks:
+            - close_channel: {from: 0, to: 1, expected_http_status: 200}
+      - parallel:
+          name: "Assert that channel closed"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "closed"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 0, state: "closed"}
+      - serial:
+          name: "Node1 tries to deposit after channel close"
+          tasks:
+            - deposit: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, expected_http_status: 409}
+      - serial:
+          name: "Send payment after channel closed"
+          tasks:
+            - transfer: {from: 0, to: 1, amount: 1_000_000_000_000_000_001, lock_timeout: 30, expected_http_status: 409}
+      - parallel:
+          name: "Assert that balances didn't change"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "closed"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 0, state: "closed"}
+      - serial: 
+          name: "Node0 tries to close the channel again"
+          tasks:
+            - close_channel: {from: 0, to: 1, expected_http_status: 409}
+      - parallel:
+          name: "Assert that nothing changed"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "closed"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 0, state: "closed"}
+      - parallel:
+          name: "Node1 tries to open the same channel before settlement"
```

Comment:
This is a funny thing to try. It's not tested. Will add a check at the same test as above.